app/knexfile.js

```diff
@@ -38,7 +41,16 @@ module.exports = {
     database: config.get('db.database'),
     port: config.get('db.port')
   },
-  debug: ['silly', 'verbose'].includes(config.get('server.logLevel')),
+  debug: ['silly', 'debug'].includes(config.get('server.logLevel')),
```

Comment:
Is there a reason we are changing the 2nd index to 'debug'?

Reply:
I guess the new verbage in the new library we are using is to just use 'debug' instead of 'verbage'?